etc/latex-common.profile

```diff
@@ -0,0 +1,36 @@
+#   NOBLACKLISTS
+noblacklist ${DOWNLOADS}
+noblacklist ${DOCUMENTS}
+noblacklist ${HOME}/texmf
+
+
+#   DISABLE INCLUDES
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
+
+#   OPTIONS (caps*, net*, no*, protocol, seccomp*, shell none, tracelog)
+caps.drop all
+net none
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix
+shell none
+seccomp
```

Comment:
```suggestion

```

Reply:
Why would you remove **seccomp**?

Reply:
See this with the commend above, https://github.com/netblue30/firejail/pull/3159#discussion_r368530574.
```
protocol unix
seccomp
shell none
```
:octocat:  has no multi-line suggestions.